add readme (#3)
AgoraIO-Usecase/AgoraChat-web: README.md

# Agora chat demo (WEB)

> This repository will help you learn how to use Agora chat SDK to realize a simple chat web application.

Reviewer: ```suggestion
This repository contains the sample project using the Agora Chat Web UI KIT.
```

> With this sample app, you can:

Reviewer: ```suggestion
With this sample project, you can:
```

> Login chat server

Reviewer: ```suggestion
1. Log in to the chat server.
```

Reviewer 2: ```suggestion
1. Log in to the chat server. 建议下面的分号都用句点，中文可以用分号，英文用句点。
```

> Start a chat

Reviewer: ```suggestion
2. Start a chat conversation.
```

> Manage session list

Reviewer: ```suggestion
3. Manage the conversation list.
```

Reviewer 2: ```suggestion
Manage the conversation list.
```

> Add friends

Reviewer: ```suggestion
4. Add friends.
```

> Join group chats

Reviewer: ```suggestion
5. Join group chats.
```

> Add your friends to your blacklist

Reviewer: ```suggestion
6. Add your contacts to your blocklist.
```

Reviewer 2: ```suggestion
Add your friends to your blocklist.
```

> Send various types of messages, Such as: text, expression, picture, voice, file and so on

Reviewer: ```suggestion
7. Send various types of messages. Such as: text, expression, picture, voice, file and so on.
```

Reviewer 2: ```suggestion
Send messages of such types as text, expression, picture, voice, and file.
```

> Send various types of messages, Such as: text, expression, picture, voice, file and so on

Reviewer: ```suggestion
7. Send messages of such types as text, expression, picture, voice, and file.
```

> Recall a message

Reviewer: ```suggestion
8. Recall a message.
```

> Logout

Reviewer: ```suggestion
9. Log out of the chat server.
```

Reviewer 2: ```suggestion
Log out of the chat server.
```

> ## Running the App

Reviewer 2: ```suggestion
## Run the App
```

``` bash

> # install dependency

Reviewer: ```suggestion
# Install dependency.
```

Reviewer 2: ```suggestion
# Install dependencies.  是安装多个依赖项？
```

npm install

> # run in dev mode

Reviewer: ```suggestion
# Run in dev mode.
```

HTTPS=true npm start

> # generate dist

Reviewer: ```suggestion
# Generate dist.
```

Reviewer 2: ```suggestion
# Generate dist files 还是  Generate the dist folder?
```

npm run build
```

## Contact Us

> - You can find full API document at[Document Center](https://hyphenateinc.github.io/web_product_overview.html).

Reviewer: ```suggestion
- You can find the complete API document at [Document Center](https://hyphenateinc.github.io/web_product_overview.html).
```

> - You can file bugs about this demo a[issue](https://github.com/AgoraIO-Usecase/AgoraChat-web/issues).

Reviewer 2: ```suggestion
- You can file bugs about this demo at [issue](https://github.com/AgoraIO-Usecase/AgoraChat-web/issues).
```

## License

> The MIT License (MIT).

Reviewer 2: ```suggestion
The MIT license.
```
